community-generators/prisma-dbml-generator/dbml/schema.dbml

```diff
@@ -0,0 +1,9 @@
+//// ------------------------------------------------------
+//// THIS FILE WAS AUTOMATICALLY GENERATED (DO NOT MODIFY)
+//// ------------------------------------------------------
+
+Table User {
+  id String [pk]
+  email String [unique, not null]
+  name String
+}
```

Comment:
Should this be committed when this is effectively what is compared against in the test? The generator not doing anything would still lead to success this way I guess.

Reply:
Good point, I can remove it at the beginning of the workflow. 

If `generate` fails the exit code should be non-zero and that should fail the test.